docs/lang/articles/gui/gui_system.md

```diff
@@ -94,26 +94,27 @@ pos = np.random.random((50, 2))
 # 1 corresponds to 0xED553B
 # 2 corresponds to 0xEEEEF0
 indices = np.random.randint(0, 2, size=(50,))
-gui = ti.GUI("lines", res=(400, 400))
+gui = ti.GUI("circles", res=(400, 400))
 while gui.running:
-    gui.circles(pos, radius=1.5, palette=[0x068587, 0xED553B, 0xEEEEF0], palette_indices=indices)
+    gui.circles(pos, radius=5, palette=[0x068587, 0xED553B, 0xEEEEF0], palette_indices=indices)
     gui.show()
 ```
 
-
-
-![img](https://o9ixctz0o7.feishu.cn/space/api/box/stream/download/asynccode/?code=N2Y2M2YxMWZlZGM2NGM1ZGQ3Y2JlZmU3NzgyMjRmZWRfSEprR1JLempxWkVSOFllRnBkS1hMZGlGeFI3SWVNN0RfVG9rZW46Ym94Y25EZFhiUmwzeWxjakllME1SSXhNMjlnXzE2NDg2MzAzNjA6MTY0ODYzMzk2MF9WNA)
+![gui-circles](https://raw.githubusercontent.com/taichi-dev/public_files/master/taichi/doc/gui-circles.png)
```

Comment:
Although the `feishu` link is broken on my end, I'm not sure if https://raw.githubusercontent.com/ is always accessible in China...

Reply:
However, other images used in taichi.dev are all using raw.githubusercontent.com, for example,
```
docs/lang/articles/contribution/compilation.md:![image](https://raw.githubusercontent.com/taichi-dev/public_files/fa03e63ca4e161318c8aa9a5db7f4a825604df88/taichi/life_of_kernel.png)
docs/lang/articles/advanced/sparse.md:![BVH](https://raw.githubusercontent.com/taichi-dev/public_files/master/taichi/doc/bvh.png)
docs/lang/articles/advanced/sparse.md:![Pointer](https://raw.githubusercontent.com/taichi-dev/public_files/master/taichi/doc/pointer.png)
docs/lang/articles/misc/internal.md:![image](https://raw.githubusercontent.com/taichi-dev/public_files/fa03e63ca4e161318c8aa9a5db7f4a825604df88/taichi/data_structure_organization.png)
```

Reply:
I see. Then it's totally fine to use it.